packages/neuron-ui/src/containers/Providers/index.tsx

```diff
@@ -89,11 +91,39 @@ const withProviders = (Comp: React.ComponentType) => (props: React.Props<any>) =
     },
   )
 
+  UILayer.on(Channel.GetWalletNameListStore, (_e: Event, args: Response<any>) => {
```

Comment:
UI layer should not have knowledge of wallet store or wallet name store. It just ask for a list of wallet objects. The store is an implementation detail in the `neuron-wallet` module.